utils/networking.js

```diff
@@ -137,6 +144,12 @@ const sendOnClickMetric = async (spaceId) => {
   try {
     const spaceClickEndpoint = API_BASE + `/api/v1/space/click/${spaceId}`
     await axios.put(spaceClickEndpoint)
+
+    await axios.post(
+      BEACON_GRAPHQL_URI,
+      { query: `mutation { increase(eventType: clicks, spaceId: "${spaceId}") { message } }` },
+      { headers: { 'Content-Type': 'application/json' }}
+    )
```

Comment:
We will be sending both beacon and beacon v2 events for now. Will be removing the old beacon after confirming that the beacon v2 works.

Reply:
<img width="1018" alt="Screen Shot 2022-02-16 at 8 18 07 PM" src="https://user-images.githubusercontent.com/93493166/154385639-464e8e15-9668-4c0f-b694-aa2e16a48ed5.png">